### PR TITLE
Car dao

### PR DESCRIPTION
**Pull request by andrlos**
initial sketch of CarDao implementation, cant think of other method, that might be usable there, you might prove me wrong during the review

### CODE REVIEW

File: src/main/java/cz/muni/pa165/teamwhite/formula1/dao/CarDao.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+    /**
+     * This method finds car in DB by entered name
+     * @param driver - driver of the requested car
+     * @return found car
+     */
+    Car findByDriver(Driver driver);
+
+    /**
+     * This method finds car in DB by entered login
```
Comment (tomasedl) [inline src/main/java/cz/muni/pa165/teamwhite/formula1/dao/CarDao.java:40]: find by entered login?

**Review by tomasedl — COMMENTED** [review]
jen asi ta vec u javadocu v interfacu

File: src/main/java/cz/muni/pa165/teamwhite/formula1/dao/CarDao.java (hunk 1 above)
Comment (andrlos) [reply src/main/java/cz/muni/pa165/teamwhite/formula1/dao/CarDao.java:40]: oh.... yeah.. i copied it from user :D

Comment (andrlos) [reply src/main/java/cz/muni/pa165/teamwhite/formula1/dao/CarDao.java]: done

**Review by Jakub-Fajkus — APPROVED** [review]

**Review by tomasedl — APPROVED** [review]

**Review by khecova — APPROVED** [review]
ok

**Pull request merged**